modules/configuring-firewall.adoc

```diff
@@ -165,6 +169,10 @@ that is specified in the `spec.route.hostname` field of the
 |`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
 |443, 80
 |Required for Sonatype Nexus, F5 Big IP operators.
+
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
```

Comment:
and I suggest moving this row up one line, close to the `registry.connect.redhat.com`

Reply:
Addressed